guides/common/modules/proc_configuring-capsule-default-certificate.adoc

```diff
@@ -44,7 +44,6 @@ _output omitted_
 --foreman-proxy-trusted-hosts "_{smartproxy-example-com}_" \
 --foreman-proxy-oauth-consumer-key "_s97QxvUAgFNAQZNGg4F9zLq2biDsxM7f_" \
 --foreman-proxy-oauth-consumer-secret "_6bpzAdMpRAfYaVZtaepYetomgBVQ6ehY_" \
---puppet-server-foreman-url "https://_{foreman-example-com}_"
```

Comment:
This one is correct, because it assumes the default scenario where Puppet is off.